exonum_client/client.py

```diff
@@ -41,17 +43,28 @@ class Subscriber:
     # Type of the `Callback` (`Callable` that takes `ReceiveType` as an argument and produces nothing).
     CallbackType = Callable[[ReceiveType], None]
 
-    def __init__(self, address: str, port: int):
+    def __init__(self, address: str, port: int, subscription_type: str, filters: Optional[Dict[str, Any]] = None):
```

Comment:
Does it make sense to add `Enum` for variants of subscription types ? Or now it's redundant ?
cc @popzxc

Reply:
I think it's fine to use string just to not bother users with additional type. Usually python users don't really love typing in places where it can be avoided, so I think we shouldn't enforce our style for everybody :)